test_runner/build.gradle.kts

```diff
@@ -31,6 +31,13 @@ shadowJar.apply {
     manifest {
         attributes(mapOf("Main-Class" to "ftl.Main"))
     }
+    dependencies {
```

Comment:
We have a dependency block at L#186, I assume this is intentional?

Reply:
Yes, it appears that if i put it at L186 it is not working as intended. Hence why it is put here. However with the changes coming with the single gradle project it most likely wont be needed anymore.

Reply:
Interesting, might be due too the fact that Gradle process script from top to bottom?